ara/ui/static/css/bootstrap-darkly.min.css

```diff
@@ -0,0 +1,12 @@
375a7f}.table-secondary,.table-secondary>td,.table-secondary>th{background-color:#444}.table-light,.table-light>td,.table-light>th{background-color:#adb5bd}.table-dark,.table-dark>td,.table-dark>th{background-color:#303030}.table-success,.table-success>td,.table-success>th{background-color:#00bc8c}.table-info,.table-info>td,.table-info>th{background-color:#3498db}.table-danger,.table-danger>td,.table-danger>th{background-color:#e74c3c}.table-warning,.table-warning>td,.table-warning>th{background-color:#f39c12}.table-active,.table-active>td,.table-active>th{background-color:rgba(0,0,0,.075)}.table-hover .table-primary:hover,.table-hover .table-primary:hover>td,.table-hover .table-primary:hover>th{background-color:#2f4d6d}.table-hover .table-secondary:hover,.table-hover .table-secondary:hover>td,.table-hover .table-secondary:hover>th{background-color:#373737}.table-hover .table-light:hover,.table-hover .table-light:hover>td,.table-hover .table-light:hover>th{background-color:#9fa8b2}.table-hover .table-dark:hover,.table-hover .table-dark:hover>td,.table-hover .table-dark:hover>th{background-color:#232323}.table-hover .table-success:hover,.table-hover .table-success:hover>td,.table-hover .table-success:hover>th{background-color:#00a379}.table-hover .table-info:hover,.table-hover .table-info:hover>td,.table-hover .table-info:hover>th{background-color:#258cd1}.table-hover .table-danger:hover,.table-hover .table-danger:hover>td,.table-hover .table-danger:hover>th{background-color:#e43725}.table-hover .table-warning:hover,.table-hover .table-warning:hover>td,.table-hover .table-warning:hover>th{background-color:#e08e0b}.table-hover .table-active:hover,.table-hover .table-active:hover>td,.table-hover .table-active:hover>th{background-color:rgba(0,0,0,.075)}.input-group-addon{color:#fff}.nav-pills .nav-item.open .nav-link,.nav-pills .nav-item.open .nav-link:focus,.nav-pills .nav-item.open .nav-link:hover,.nav-pills .nav-link,.nav-pills .nav-link.active,.nav-pills .nav-link.active:focus,.nav-pills .nav-link.active:hover,.nav-tabs .nav-item.open .nav-link,.nav-tabs .nav-item.open .nav-link:focus,.nav-tabs .nav-item.open .nav-link:hover,.nav-tabs .nav-link,.nav-tabs .nav-link.active,.nav-tabs .nav-link.active:focus,.nav-tabs .nav-link.active:hover{color:#fff}.breadcrumb a{color:#fff}.pagination a:hover{text-decoration:none}.close{opacity:.4}.close:focus,.close:hover{opacity:1}.alert{border:none;color:#fff}.alert .alert-link,.alert a{color:#fff;text-decoration:underline}.alert-primary{background-color:#375a7f}.alert-secondary{background-color:#444}.alert-success{background-color:#00bc8c}.alert-info{background-color:#3498db}.alert-warning{background-color:#f39c12}.alert-danger{background-color:#e74c3c}.alert-light{background-color:#adb5bd}.alert-dark{background-color:#303030}.list-group-item-action{color:#fff}.list-group-item-action:focus,.list-group-item-action:hover{background-color:#444;color:#fff}.list-group-item-action .list-group-item-heading{color:#fff}
```

Comment:
TODO: just noticed this: ``/@import url(https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400&display=swap);``. I'd much rather have the font self hosted and it doesn't seem too complicated according to this issue: https://github.com/thomaspark/bootswatch/issues/573

Reply:
Just removing the import seems to be sufficient without any discernable impact that I can see -- the font appears to be the same and we're not calling out to google anymore.

Before:
![Screenshot from 2021-03-17 14-38-00](https://user-images.githubusercontent.com/1291204/111520453-7e2c2600-872e-11eb-9ceb-fa628b3ab91d.png)

After:
![Screenshot from 2021-03-17 14-37-14](https://user-images.githubusercontent.com/1291204/111520467-82f0da00-872e-11eb-8926-42863233a6b9.png)